deploy/docker-compose.yml

```diff
@@ -41,13 +41,14 @@ services:
       - 42114:42114/tcp
 
   db:
-    build:
-      context: db
+    image: postgres:10-alpine
     restart: unless-stopped
     environment:
       POSTGRES_DB: tinkerbell
       POSTGRES_PASSWORD: tinkerbell
       POSTGRES_USER: tinkerbell
+    volumes:
+      - ./db/tinkerbell-init.sql:/docker-entrypoint-initdb.d/tinkerbell-init.sql:ro
```

Comment:
I'm not actually finding a local folder called `db` in the deploy folder after running setup.sh?

<img width="1077" alt="Screenshot 2020-05-18 at 14 10 48" src="https://user-images.githubusercontent.com/6358735/82216797-666af200-9911-11ea-9199-4c4f40564ebf.png">

Reply:
Because setup.sh fetches from master. I commented out the git fetch bit and used my own clone of this pr and it worked out fine.